redbot/cogs/mod/slowmode.py

```diff
@@ -0,0 +1,29 @@
+from redbot.core import commands, i18n, checks
+from .abc import MixinMeta
+
+_ = i18n.Translator("Mod", __file__)
+
+
+class Slowmode(MixinMeta):
+    """
+    Commands regarding channel slowmode management.
+    """
+
+    @commands.command()
+    @commands.guild_only()
+    @commands.bot_has_permissions(manage_channels=True)
+    @checks.mod_or_permissions(manage_channels=True)
+    async def slowmode(self, ctx, interval: int = 0):
+        """Change channel's slowmode setting.
+
+        Interval can be anything from 0 to 21600 seconds.

```

Comment:
can you add human time format converter?

seconds, minutes, hours etc

Reply:
As the input parameter? Or for the `Slowmode interval is now {interval} seconds.` feedback message?

Reply:
both

Reply:
Should be resolved with the recent commit.